### PR TITLE
Feature/asep 58

### PR DESCRIPTION
**Pull request by saurabhprasun20**
Changes:
- Swagger API changes
-  javadoc related build.gradle changes (Adding the plugin)
- README.md file changes for Airbnb-API

### CODE REVIEW

File: eloquent-elks-airbnb-api/src/main/java/com/airbnb/eloquentelksbackend/repository/PropertyRepository.java [hunk 1]
```diff
@@ -16,12 +16,15 @@
...
     /**
-     * findAll method returns all the Property present in the database
+     * 
+     * @param north Latitude of the bounding box in the Northern edge
+     * @param east Longitude of the bounding box in the Eastern edge
+     * @param south Latitude of the bounding box in the Northern edge
+     * @param west Longitude of the bounding box in the Western edge
+     * @param minPrice minPrice filter to fetch the properties above the minPrice
+     * @param maxPrice maxPrice filter to fetch the properties below the maxPrice
      * @return List of Properties @see {@link com.airbnb.eloquentelksbackend.entity.Property}
```
Comment (niels89) [inline eloquent-elks-airbnb-api/src/main/java/com/airbnb/eloquentelksbackend/repository/PropertyRepository.java:26]: We should remove the @see references, as they are not interpreted together with the @link statements.

**Review by niels89 — COMMENTED** [review]
We should check if we have reduntant @see statements in the javadoc comments, so it won't impair the visual representation.

File: eloquent-elks-airbnb-api/src/main/java/com/airbnb/eloquentelksbackend/repository/PropertyRepository.java (hunk 1 above)
Comment (saurabhprasun20) [reply eloquent-elks-airbnb-api/src/main/java/com/airbnb/eloquentelksbackend/repository/PropertyRepository.java:26]: Done.

**Review by niels89 — APPROVED** [review]
Nice work! Looks good to me!

**Pull request merged**